tests/test_oauth1_session.py

```diff
@@ -95,6 +109,11 @@ def fake_send(r, **kwargs):
     @mock.patch('oauthlib.oauth1.rfc5849.generate_timestamp')
     @mock.patch('oauthlib.oauth1.rfc5849.generate_nonce')
     def test_signature_methods(self, generate_nonce, generate_timestamp):
+        try:
+            import cryptography
+        except ImportError:
+            raise unittest.SkipTest('cryptography not available')
```

Comment:
I've moved the import at the top.
Unfortunately skipUnless isn't available on Python 2.6's `unittest`, and using monkey patching for that is going too far IMO.  We could switch to using `unittest2` on Python 2.6.